src/lib/components/event/pending-activity-summary-row.svelte

```diff
@@ -1,46 +1,72 @@
 <script lang="ts">
   import { page } from '$app/stores';
 
+  import Badge from '$lib/holocene/badge.svelte';
+  import Copyable from '$lib/holocene/copyable/index.svelte';
   import Icon from '$lib/holocene/icon/icon.svelte';
   import Link from '$lib/holocene/link.svelte';
   import { translate } from '$lib/i18n/translate';
   import type { EventGroup } from '$lib/models/event-groups/event-groups';
+  import { isCloud } from '$lib/stores/advanced-visibility';
+  import { relativeTime, timeFormat } from '$lib/stores/time-format';
   import type { PendingActivity } from '$lib/types/events';
+  import { formatDate } from '$lib/utilities/format-date';
   import { routeForEventHistoryEvent } from '$lib/utilities/route-for';
   import { toTimeDifference } from '$lib/utilities/to-time-difference';
 
   import EventDetailsFull from './event-details-full.svelte';
   import EventDetailsRow from './event-details-row.svelte';
 
-  export let event: PendingActivity;
-  export let group: EventGroup | undefined = undefined;
-  export let index: number;
-  export let expandAll = false;
-  export let active = false;
-  export let onRowClick: () => void = () => {};
+  interface Props {
+    event: PendingActivity;
+    group: EventGroup | undefined;
+    index: number;
+    expanded?: boolean;
+    onRowClick?: () => void;
+  }
+
+  const {
+    event,
+    group,
+    index,
+    expanded: expandedProp = false,
+    onRowClick = () => {},
+  }: Props = $props();
 
-  $: expanded = expandAll;
-  $: ({ workflow, run, namespace } = $page.params);
-  $: href = routeForEventHistoryEvent({
-    eventId: group?.id,
-    namespace,
-    workflow,
-    run,
-  });
+  let expanded = $state(expandedProp);
+  let { workflow, run, namespace } = $derived($page.params);
```

Comment:
If you use `page` from `$app/state` it's already reactive and `$derived` likely isn't required.